account_payment_term_partner_holiday/tests/test_partner_holiday.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import datetime
+from odoo.tests import common
+from odoo.tests.common import Form
+
+
+class TestPartnerHoliday(common.TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.partner_1 = self.env["res.partner"].create({"name": "Partner test 1"})
+        self.partner_2 = self.env["res.partner"].create({"name": "Partner test 2"})
+        self.partner_1.write({
```

Comment:
Why not putting this in the creation above? If there is this particular need, maybe put a comment here explaining it to make it clearer...